CONTRIBUTING.md

```diff
@@ -0,0 +1,181 @@
+# Contributing
+
+For documentation changes, you can just edit and make a pull request. Ignore the instructions below.
+
+## Setup
+
+1. Asking to join as a contributor is simplest. You can fork, but you will have to change lines in your `.env` file and may have to set GitHub secrets in your repository. You have to talk to us either way.
+1. Clone the repository locally.
+1. Run `npm install`
+1. Create your `.env` file at the root of the project.
+1. Contact @plocket to get important information for your `.env` file.
+1. Run `npm run setup`
+
+Advanced contributors: If you're going to work on our docassemble internal testing interviews, do so as a contributor on this repository. Avoid forking if you can. The reasons are complex.
+
+## PRs
+
+Before making a pull request
+
+1. Search issues to see if what you are thinking of has been discussed yet.
+1. Make an issue about the change you are thinking of.
+1. Assign yourself to an issue.
+1. Make a new branch.
+1. Make your edits. Feel free to run tests as you go.
+1. When you're ready to push
+   1. Run `npm run test` to make sure your code passes all our tests.
+   1. Make an entry in the `CHANGELOG.md` describing your change.
+   1. AVOID changing the version number. We'll do that after merging your changes.
+   1. If needed, update the `README.md` (e.g. the cheat sheet section) and/or `CONTRIBUTING.md` docs. Changes to our architecture might require editing `CONTRIBUTING.md`.
+1. Push to GitHub.
+1. Wait for the GitHub tests to pass. You may have to re-run failing tests - server restarts or race conditions may interfere with them. We have reduced them greatly, but some are impossible to avoid.
```

Comment:
Feedback:
```suggestion
1. Wait for the GitHub tests to pass. You may have to re-run failing tests—server restarts or race conditions may interfere with them. We have reduced them greatly, but some are impossible to avoid.
```